### PR TITLE
Add Safari settings guide for adguard to take effect

### PR DESCRIPTION
**Pull request by ETOgaosion**
Add Safari settings guide for adguard to take effect, hope to guide others to avoid common issue.

Here is preview image of modified contents:

截屏2023-10-27 17 25 21

### CODE REVIEW

**Comment by ETOgaosion** [comment]
You can download the pictures with the image bed url I use in the file, put them on your cdn host, and rename it as commented

**Comment by ETOgaosion** [comment]
@ameshkov Hello Mr. would u consider my edition~

File: docs/adguard-for-safari/installation.md [hunk 1]
```diff
@@ -21,17 +21,37 @@ AdGuard for Safari is a free app presented on the App Store. To install it on yo
...
 
-Not all extensions are required for AdGuard for Safari to function correctly. For instance, checking the *Icon* extension enables the icon next to the search bar. However, we recommend enabling all of them.
+    Not all extensions are required for AdGuard for Safari to function correctly. For instance, checking the *Icon* extension enables the icon next to the search bar. However, we recommend enabling all of them.
 
-:::
+    :::
+
+    ![Extensions checked](https://cdn.adtidy.org/content/Kb/ad_blocker/safari/adguard-for-safari-extensions-checked.png)
+
+1. To make Adguard take effects on websites, *content blocker* setting must be enabled. you can either enable for all website or select those ones you want.
```
Comment (el-termikael) [inline docs/adguard-for-safari/installation.md:34]:
```suggestion
1. To make AdGuard take effects on websites, the *Content Blockers* setting must be enabled. You can either enable for all websites or select the ones you want.

File: docs/adguard-for-safari/installation.md [hunk 2]
```diff
@@ -21,17 +21,37 @@ AdGuard for Safari is a free app presented on the App Store. To install it on yo
...
+
+    :::note
+
+    This step is needed for iOS 9.0 and later, macOS 10.11 with Safari 10, and macOS 10.12 and later.
+
+    :::
+
+    Here is where to set *content blocker* on MacOS Safari: click *Preference* → *Websites* → *Content Blockers*, if you want to enable adguard on every website, turn *When visiting other websites* to *on*.
```
Comment (el-termikael) [inline docs/adguard-for-safari/installation.md:42]:
```suggestion
On macOS, click *Preference* → *Websites* → *Content Blockers*. If you want to enable AdGuard on every website, turn *When visiting other websites* to *On*.

File: docs/adguard-for-safari/installation.md [hunk 3]
```diff
@@ -21,17 +21,37 @@ AdGuard for Safari is a free app presented on the App Store. To install it on yo
...
+
+    ![Mac Safari Content Blocker Setting](https://i0.imgs.ovh/2023/10/26/Fmc9U.png)
+
+    <!-- adguard-for-safari-content-blocker-setting-macos.png -->
+
+    As for iOS (take ipad pro for example), go *Settings* → *Safari* → *Content Blockers*, turn option to *yes*.
```
Comment (el-termikael) [inline docs/adguard-for-safari/installation.md:48]:
```suggestion
On iOS, go *Settings* → *Safari* → *Content Blockers*, and set the option to *Yes*.

**Review by el-termikael — CHANGES_REQUESTED** [review]
@ETOgaosion thank you for adding Safari settings guide! please make suggested changes and we'll be able to merge it into master

**Comment by ETOgaosion** [comment]
Yeah, detail issues, my language mistakes(qaq).

Now accepted all your suggestions~

**Review by el-termikael — APPROVED** [review]

**Pull request merged**